app/server/appsmith-server/src/main/java/com/appsmith/server/constants/FieldName.java

```diff
@@ -118,4 +118,6 @@ public class FieldName {
     public static final String AUDIT_DATA = "auditData";
     public static final String MODE_OF_LOGIN = "modeOfLogin";
     public static final String FORM_LOGIN = "FormLogin";
+    public static final String VIEW_MODE = "viewMode";
```

Comment:
```suggestion
    public static final String APP_MODE = "appMode";
```
Suggestion to rename the variable to accurately reflect what we are signifying.